modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js

```diff
@@ -61,3 +93,89 @@ test('Become user', async(t) => {
         // See https://ggsystems.atlassian.net/browse/GG-24068
         .expect(userMenu.menuItem.withText('Admin panel').count).eql(1);
 });
+
+test("Action availability", async(t) => {
+    const _changeUserSelection = async(userName) => {
+        await t.click(admin.userNameCell.withText(userName))
+            .hover(admin.usersTable.actionsButton);
+    };
+
+    await t.navigateTo(resolveUrl('/settings/admin'))
+        .hover(admin.usersTable.actionsButton);
+
+    await _checkActionSet(t, true, true);
+
+    await admin.usersTable.performAction('Add user');
+
+    const createUserDialog = Selector('.modal-dialog');
+
+    await t.expect(createUserDialog.find('.modal-header').withText('Create User').exists).ok()
+        .click(createUserDialog.find('button').withText('Cancel'))
+        .hover(admin.usersTable.actionsButton);
+
+    await _checkActionSet(t, true, true);
+
+    await _changeUserSelection(CUR_USER);
+
+    await _checkActionSet(t, true, false);
+
+    await _changeUserSelection(CUR_USER);
+
+    await _checkActionSet(t, true, true);
+
+    await _changeUserSelection(OTHER_USER);
+
+    await _checkActionSet(t, false, false);
+
+    await _changeUserSelection(OTHER_USER);
+
+    await _checkActionSet(t, true, true);
+
+    await _changeUserSelection(OTHER_USER);
+
+    await admin.usersTable.performAction('Revoke admin');
+
+    await t.expect(successNotification.withText(`Admin rights was successfully revoked for user: "${OTHER_USER}"`).exists).ok();
+
+    await t.eval(() => window.location.reload());
+    await t.expect(admin.userNameCell.withText(OTHER_USER).find('i.icon-user').exists).ok(LONG_TIMEOUT_OPTION);
+
+    await _changeUserSelection(OTHER_USER);
+    await admin.usersTable.performAction('Grant admin');
+
+    await t.expect(successNotification.withText(`Admin rights was successfully granted for user: "${OTHER_USER}"`).exists).ok();
+
+    await t.eval(() => window.location.reload());
+    await t.expect(admin.userNameCell.withText(OTHER_USER).find('i.icon-admin').exists).ok(LONG_TIMEOUT_OPTION);
```

Comment:
Why lines 141 and 149 are equal? In 1st case user shouldn't be an admin and vice versa in 2nd case